server/utils/user-stats.js

```diff
@@ -2,7 +2,7 @@ import _ from 'lodash';
 import moment from 'moment-timezone';
 import { dayCount } from '../utils/date-utils';
 
-const daysBetween = 1.5;
+const daysBetween = 2;
```

Comment:
Changed this to two because it ill now reference a full 48 hours. Everything that uses this constant compares it (less than or greater than) so it shouldn't break what streak functionality but add the time that was missed.